### ISSUE TITLE
[REQUEST] add "timestamp" parameter instead of "7:30 PM ET"

### ISSUE THREAD

**Issue by CyrilNb** [ynnadkrap/balldontlie#100]
Hello, 

I live in Europe and it would be more convenient to have timestamp parameter in addition to "7:30 PM ET" as it's hard to convert into any other timezone.

Thanks a lot!

**Comment by treyg** [comment]
If you're using JS, you could just use a library like day.js to convert to whatever format you need. I've used it a decent amount if you have any questions. Moment JS is also good, but I think it may be deprecated

https://day.js.org/

**Comment by CyrilNb** [comment]
No I am not using JS. And it's still better to have a universal format like a timestamp

**Comment by Ptujec** [comment]
+1 
I think it would be the best to include the correct time in the date parameter.

**Comment by CyrilNb** [comment]
@ynnadkrap would it be possible? Thank you so much. I'm struggling with the date and time format the API is sending

**Comment by aberaldo** [comment]
Yes the best is a timestamp including the our. And include in the date attribute. Because in status then we dont know at which our was the date when the game ends. Eg

![image](https://user-images.githubusercontent.com/8792209/235254677-8dd3d288-a43c-4f9c-8229-fae95a22f5fa.png)

When the status is "Final" we lost the our of previous games
